examples/glyphs/airports_map.py

```diff
@@ -42,12 +38,4 @@
 points_glyph = Circle(**point_options)
 p.add_glyph(points_source, points_glyph)
 
-doc = Document()
-doc.add_root(p)
-
-if __name__ == "__main__":
-    filename = "airports_map.html"
-    with open(filename, "w") as f:
-        f.write(file_html(doc, INLINE, "Bokeh Airports Example"))
-    print("Wrote %s" % filename)
-    view(filename)
```

Comment:
`examples/glyphs` use the verbose API, so this should be restored. Otherwise you can move it to `examples/plotting/file`. Actually, it used to use a mixture of both, so just do the later.